### PR TITLE
Fix menu and game flow

### PR DESCRIPTION
**Pull request by BartoszChadrys**

### CODE REVIEW

**Review by Bzygu234 — COMMENTED** [review]
We can set width, height and coordinates as constants

File: Menu/Menu.py [hunk 1]
```diff
@@ -116,29 +116,97 @@ def drawAllPlayersInLobby(self):
...
         self.drawCircles()
-
         self.drawAllPlayersInLobby()
 
-        pygame.display.update()
+    def showLobby(self):
+        self.showLobbyBackground()
+        mousePos = pygame.mouse.get_pos()
+
+        backButton = Button(pos=(CENTER_X_POS, 640), textInput="BACK", font=getFont(50),
```
Comment (Bzygu234) [inline Menu/Menu.py:131]: We can set screen width, height and coordinates as constants

File: Menu/Menu.py [hunk 2]
```diff
@@ -116,29 +116,97 @@ def drawAllPlayersInLobby(self):
...
+    def drawTextField(self, textFieldRect, color):
+        pygame.draw.rect(self.screen, color, textFieldRect)
+
+    def drawInput(self, text):
+        inputText = getFont(13).render(text, True, "black")
+        inputRect = inputText.get_rect(center=(CENTER_X_POS, 330))
+        self.screen.blit(inputText, inputRect)
+
+    def showJoinScreen(self):
+        textField = pygame.Rect(CENTER_X_POS-150, 305, 300, 50)
```
Comment (Bzygu234) [inline Menu/Menu.py:154]: Same here

File: Menu/Menu.py [hunk 3]
```diff
@@ -116,29 +116,97 @@ def drawAllPlayersInLobby(self):
...
+        input = ''
+        textFieldActive = False
+        joinScreenRunning = True
+
+        joinButton = Button(pos=(CENTER_X_POS, 640), textInput="JOIN", font=getFont(50),
```
Comment (Bzygu234) [inline Menu/Menu.py:159]: And here

**Review by Bzygu234 — COMMENTED** [review]
We can set width, height and coordinates as constants

File: Menu/Menu.py (hunk 1 above)
Comment (BartoszChadrys) [reply Menu/Menu.py]: Done

File: Menu/Menu.py (hunk 2 above)
Comment (BartoszChadrys) [reply Menu/Menu.py]: Done

File: Menu/Menu.py (hunk 3 above)
Comment (BartoszChadrys) [reply Menu/Menu.py]: Done

**Review by Bzygu234 — APPROVED** [review]

**Review by olekgolus11 — APPROVED** [review]
g

**Pull request merged**